### PR TITLE
{bio}[foss/2022a] BAMM v2.5.0

### PR DESCRIPTION
**Pull request by btlindert**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/364242cebdd02241083aaee18ef908ee for a full test report.

File: easybuild/easyconfigs/b/BAMM/BAMM-2.5.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of 
+lineage diversification and trait evolution. """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': True}
+
+# https://github.com/macroevolution/bamm
+github_account = 'macroevolution'
+# source_urls = ['https://github.com/macroevolution/bamm/archive']
+source_urls = ['GITHUB_LOWER_SOURCE']
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BAMM/BAMM-2.5.0-foss-2022a.eb:17]:
```suggestion
source_urls = [GITHUB_LOWER_SOURCE]
```

**Review by verdurin — DISMISSED** [review]

File: easybuild/easyconfigs/b/BAMM/BAMM-2.5.0-foss-2022a.eb (hunk 1 above)
Comment (btlindert) [reply easybuild/easyconfigs/b/BAMM/BAMM-2.5.0-foss-2022a.eb]: Thanks, fixed!

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/71f149245d0f7c05d0ba980964a87cbe for a full test report.

**Comment by Micket** [comment]
Going in, thanks @btlindert!

**Pull request merged**